fourkitchens/suhumsci
# Shs 4862 toolbar shortcuts

## Summary
This adds a module that adjusts toolbar permissions for user roles.

## Steps to Test
1. After downloading the branch and logging in as normal, make sure you can still see the admin toolbar and shortcuts as normal.
2. Go through each role below 'developer' (Site Manager, etc.) and ensure you cannot see the admin toolbar and **only** the shortcuts toolbar.
3. Anything less than 'Contributor' and you won't see a toolbar
4. Do this on a separate branch to ensure that the desired outcome is the same - ie an 'Author' has the same experience because they won't see any toolbar - this is an assumption, see the note below.

## Notes
- For points 3/4 above, we should verify that this is the expected behavior. I'm not sure how an 'Author' edits content, but when logging in as an Author in a separate branch, I did not see a toolbar as well.
- I branched this off my HTML CK branch for ease of development. 

## Screenshots
Site Manager
![siteManager](https://user-images.githubusercontent.com/8405274/232852806-12d73d49-4c72-4f52-a2ba-43bfd41f1868.png)

Contributor
![contributor](https://user-images.githubusercontent.com/8405274/232852828-bc73ff3f-154c-459e-84d1-1b781d429c00.png)

Author
![author](https://user-images.githubusercontent.com/8405274/232852843-6b76ee17-5529-4ce5-acb3-27e4f5dd4b4f.png)
